styleguide/derek/incubation/refresh.jade

```diff
@@ -0,0 +1,92 @@
+      li No outdated technology
+      li High Quality
+      li Full Color (not black and white)
+      li Doesn't contain any predominant non-rackspace logos
+
+.container
+.row.half-padding-full
+  .col-md-12
+    h5 Large Refresh Banner
+    p This banner should be used for headlines between 70 and 88 characters long. The image area is larger and the box width is wider to accommodate for longer headlines.
+    p.red.uppercase Do not use this banner for shorter headlines in order to have a larger banner image area. The banner is dictated by headline length/size.
+
+!=partial("../pattern-components/headers/header-refresh", {desc: "we guarantee a 99.99% openstack API uptime.<br>And yes, that .01% does keep us up at night", size: "Large"})
+
+.container
+.row.half-padding-full
+  .col-md-12
+    h5 Medium IMAC Banner
+    p This banner should be used for headlines between 50 and 70 characters long. The image area and box width is medium-sized to accommodate for moderate headlines.
+
+!=partial("../pattern-components/headers/header-refresh", {desc: "the cloud is a mystery wrapped in a riddle.<br>We can unwrap it for you", size:"Medium"})
+
+.container
+.row.half-padding-full
+  .col-md-12
+    h5 Small IMAC Banner
+    p This banner should be used for headlines under 50 characters long. The image area and box width is small to accommodate for our much shorter headlines.
+    p.red.uppercase Do not use this banner for larger headlines in order to have a smaller banner image area. The banner is dictated by headline length/size.
+
+!=partial("../pattern-components/headers/header-refresh", {desc: "The cloud runs on its own,<br>with lots of help", size:"Small"})
+
+hr
+
+.container
+.row.half-padding-full
+  .col-md-12
+    h2 Examples
+    p All the following are examples of headlines given by Y&amp;R and how they look/will be put into banner areas. If we need to remove/add buttons and button text we can make that happen. There were no examples of internal pages. Still need photography examples.
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "Certified to manage the world’s leading clouds. <br/> Because one size doesn’t fit all", size:"Medium"})
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "6 different clouds, 6 different vendors. <br/>How’s that working out for you?", size:"Medium"})
+
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "the cloud keeps you ahead of the curve, <br/>but what if it throws you a few", size:"Medium"})
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "3,000 cloud experts working for you 24/7? <br/>Sounds like you may need a bigger office", size:"Medium"})
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "1 B hours managing openstack clouds? <br/>Talk about putting time on your side", size:"Medium"})
```

Comment:
Not sure if just a content problem but just wanna sure we're aware.  This text gets cut off oddly at around 820px.

![image](https://user-images.githubusercontent.com/5263371/30127467-00bb2d68-9305-11e7-9cab-ef61d0f37612.png)